src/lib/github/profile.ts

```
@@ -0,0 +1,494 @@
+  if (!user) {
+    console.error("[GITHUB_API] No user data in response", { username });
+    throw new Error("USER_NOT_FOUND");
+  }
+  console.log("[GITHUB_API] User data parsed successfully", {
+    username: user.login,
+    repos: user.repositories.nodes.length,
+  });
+
+  const coll = user.contributionsCollection;
+  const initialRepos = user.repositories.nodes || [];
+  let allRepos: GithubRepoNode[] = [...initialRepos];
```

ESLint

[error] 'allRepos' is never reassigned. Use 'const' instead.

(prefer-const)